bedrock/firefox/templates/firefox/welcome/page2.html

```diff
@@ -0,0 +1,65 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/welcome/page2" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{{ _('Pocket - Save news, videos, stories and more') }}{% endblock %}
+
+{% block page_image %}{{ static('img/firefox/template/page-image-master.jpg') }}{% endblock %}
+
+{% block body_class %}welcome-page2{% endblock %}
+
+{% block content_intro %}
+  <h1 class="c-page-title">{{ _('Your time online is worth protecting') }}</h1>
+  {# L10n: "Pocket" is a product name and shouldn't be translated. #}
+  <p class="c-page-tagline">{{ _('Discover and save stories in Pocket — and come back to them when you’re free.') }}</p>
+{% endblock %}
+
+{% block content_primary %}
+  <h2 class="body-primary-title"><img src="{{ static('protocol/img/logos/pocket/logo-word-horz.svg') }}" width="186" height="" alt="Pocket"></h2>
+
+  <div class="body-primary-body">
+    <p>{{ _('Pocket is built right into Firefox, so you can easily save stories as you find them, then read them later on any device.') }}</p>
+
+    <img class="primary-image" src="{{ static('img/firefox/welcome/pocket-hero.svg') }}" width="700" height="" alt="">
+
+    <p class="primary-cta">
+      <a id="pocket-cta" class="mzp-c-button mzp-t-product" href="https://getpocket.com/ff_signup?s=ffwelcome2&form_type=button&entrypoint=mozilla.org-firefox-welcome-2&utm_source=mozilla.org-firefox-welcome-2&utm_campaign=welcome-2-pocket&utm_medium=referral" data-action="{{ settings.FXA_ENDPOINT }}" data-cta-text="Activate Pocket" data-cta-type="FxA-Pocket" data-cta-position="Primary" rel="external noopener">{{ _('Activate Pocket') }}</a>
```

Comment:
Is it possible for folks who do not have a firefox account to be shown this page? Afaik, I think this page is only shown to fxa users, but just wanted to double-check.

Reply:
I believe it's shown to people X days after they create a new profile, which is usually a new user so many if not most of them will not have a Firefox account. Promoting other products is an incentive to join.